tensorflow/lite/python/lite_test.py

```diff
@@ -881,9 +881,18 @@ def testPostTrainingCalibrateAndQuantize(self, enable_mlir):
     self.assertLess(len(quantized_tflite), len(float_tflite))
 
   @parameterized.named_parameters(
-      ('EnableMlirConverter', True),  # enable mlir
-      ('DisableMlirConverter', False))  # disable mlir
-  def testCalibrateAndQuantizeBuiltinInt8(self, enable_mlir):
+      # Quantize model to Int8: with enable mlir
+      ('UseTfliteBuiltinsIntEnableMLIR',
+       [lite.OpsSet.TFLITE_BUILTINS_INT8], True),
+      # Quantize model to Int8: with disable mlir
+      ('UseTfliteBuiltinsIntDisableMLIR',
+       [lite.OpsSet.TFLITE_BUILTINS_INT8], False),
+      # Quantize model to Int16: with disable mlir
+      ('UseTfliteBuiltinsInt16DisableMLIR',
+       [lite.OpsSet.\
+       EXPERIMENTAL_TFLITE_BUILTINS_ACTIVATIONS_INT16_WEIGHTS_INT8],
+       False))
```

Comment:
This is confusing to me. Why would we only have a test with MLIR disabled? MLIR is the default path, so this leaves us with no test coverage. It's the old (TOCO) path for which we don't need to guarantee compatibility. Also, is this the only end-to-end test? Seems like the test coverage is a bit thin?

Reply:
Hi @jdduke Thanks for noticing this - I updated it, so the new converter is tested as well (this test was done ages ago, when the new converter was in development - i forgot to update).
There is another PR with full coverage for this mode (I will correct an option once it will be accepted):
https://github.com/tensorflow/tensorflow/pull/39543
We agreed that it will go as a separate PR, because this one is a very big PR already.